packages/react-native/package.json

```diff
@@ -28,19 +28,20 @@
     "events": "^3.3.0"
   },
   "devDependencies": {
-    "@types/react-native": "^0.64.10",
     "@types/indy-sdk-react-native": "npm:@types/indy-sdk@^1.16.6",
+    "@types/react-native": "^0.64.10",
+    "indy-sdk-react-native": "^0.1.13",
     "react": "17.0.1",
     "react-native": "0.64.2",
-    "rimraf": "~3.0.2",
-    "typescript": "~4.3.0",
-    "indy-sdk-react-native": "^0.1.13",
+    "react-native-fs": "^2.18.0",
     "react-native-get-random-values": "^1.7.0",
-    "react-native-fs": "^2.18.0"
+    "rimraf": "~3.0.2",
+    "typescript": "~4.3.0"
   },
   "peerDependencies": {
+    "buffer": "^6.0.3",
```

Comment:
`buffer` is already a dependency of core and we use it in other places. In `WsOutboundTransport` you can do: 

```
import { Buffer } from '../utils/buffer'
```